### ISSUE TITLE
Initialization with decimal

### ISSUE THREAD

**Issue by nachogrxdev** [dinerojs/dinero.js#731]
I have been reviewing the documentation and I have not found a way to initialize the money class with a decimal amount. Maybe I missed something, because it seems strange to me that there is no initializer for this situation.

Thank you :)

**Comment by macdonaldr93** [comment]
Yeah! +1 to this :) I love this library, but it's difficult to use it with Shopify. All of Shopify's monetary values are string decimals.

**Comment by eladchen** [comment]
v2 docs: 

https://v2.dinerojs.com/docs/faq/how-can-i-create-dinero-objects-from-floats

They are outdated though... 

currency.base might be a number or an array of number :|
